stellar-contract-env-host/src/storage.rs

```diff
@@ -10,7 +10,7 @@ pub struct Key {
     pub key: ScVal,
 }
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, PartialEq)]
```

Comment:
I don't think there's a reason we can't add Eq either?
```suggestion
#[derive(Clone, Debug, Eq, PartialEq)]
```

Reply:
Fixed.